.travis.yml

```diff
@@ -6,44 +7,28 @@ php:
     - 5.6
     - 7.0
     - hhvm
-    - hhvm-nightly
 
 env:
     global:
         - TEST_SERVER=http://127.0.0.1:10000/server.php
-        - PHP_FCGI_CHILDREN=10
         - SYMFONY_VERSION=2.3.*
         - COMPOSER_PREFER_LOWEST=false
         - ENABLE_CURL=true
 
-cache:
-    apt: true
-    directories:
-        - vendor
-        - $HOME/.composer/cache
-
 install:
-    - sudo apt-get update
-    - sudo apt-get install apache2 libapache2-mod-fastcgi
-    - sudo a2enmod actions fastcgi alias
-    - if [[ "$TRAVIS_PHP_VERSION" != hhvm* ]]; then sudo cp -f tests/Fixtures/etc/apache-php.conf /etc/apache2/sites-available/default; fi
-    - if [[ "$TRAVIS_PHP_VERSION" = hhvm* ]]; then sudo cp -f tests/Fixtures/etc/apache-hhvm.conf /etc/apache2/sites-available/default; fi
-    - sudo sed -i "s?%SERVER_DIR%?$(pwd)/tests/Fixtures?g" /etc/apache2/sites-available/default
-    - sudo service apache2 restart
-    - if [[ "$TRAVIS_PHP_VERSION" != hhvm* ]]; then printf "\n" | pecl install propro; fi
-    - if [[ "$TRAVIS_PHP_VERSION" != hhvm* ]]; then printf "\n" | pecl install raphf; fi
-    - if [[ "$TRAVIS_PHP_VERSION" != hhvm* ]]; then printf "\n" | pecl install pecl_http; fi
+    - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]] && [[ "$TRAVIS_PHP_VERSION" != "7" ]]; then printf "\n" | pecl install propro; fi
+    - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]] && [[ "$TRAVIS_PHP_VERSION" != "7" ]]; then printf "\n" | pecl install raphf; fi
+    - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]] && [[ "$TRAVIS_PHP_VERSION" != "7" ]]; then printf "\n" | pecl install pecl_http; fi
     - if [[ "$ENABLE_CURL" = false ]]; then mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d; fi
     - if [[ "$ENABLE_CURL" = false ]]; then echo "disable_functions = curl_init" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
     - if [[ "$TRAVIS_PHP_VERSION" = "5.6" ]]; then echo "always_populate_raw_post_data = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
-    - if [[ "$TRAVIS_PHP_VERSION" != hhvm* ]]; then php-cgi -b 127.0.0.1:9000 & fi
-    - if [[ "$TRAVIS_PHP_VERSION" = hhvm* ]]; then hhvm --mode server -vServer.Type=fastcgi -vServer.Port=9000 -vServer.FixPathInfo=true & fi
-    - composer self-update
+    - php -S 127.0.0.1:10000 -t tests/Fixtures > /dev/null 2>&1 &
     - composer require symfony/event-dispatcher:${SYMFONY_VERSION} --no-update --dev
+    - if [[ $COMPOSER_PREFER_LOWEST = true ]]; then composer remove guzzlehttp/ringphp --no-update --dev; fi
     - if [[ "$SYMFONY_VERSION" = *dev* ]]; then sed -i "s/\"MIT\"/\"MIT\",\"minimum-stability\":\"dev\"/g" composer.json; fi
     - composer update --prefer-source `if [[ $COMPOSER_PREFER_LOWEST = true ]]; then echo "--prefer-lowest --prefer-stable"; fi`
```

Comment:
This is not a very nice solution. I suggest checking the config in the new org packages.